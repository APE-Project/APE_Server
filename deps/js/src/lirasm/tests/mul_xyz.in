; 1073741823 * 2 < 2^31, and will not overflow.
big = immi 1073741823
two = immi 2

res = muli big two

; Ensure that 'big' and 'two' get their own registers and
; aren't shared with 'res'.
m = allocp 12
sti big m 0
sti two m 4

reti res
