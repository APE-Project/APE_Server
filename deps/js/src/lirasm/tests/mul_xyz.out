Output is: 2147483646
